components/brave_wallet/browser/blockchain_registry.cc

```diff
@@ -58,11 +58,10 @@ mojom::BlockchainTokenPtr BlockchainRegistry::GetTokenByAddress(
     return nullptr;
 
   const auto& tokens = token_list_map_[key];
-  auto token_it =
-      std::find_if(tokens.begin(), tokens.end(),
-                   [&](const mojom::BlockchainTokenPtr& current_token) {
-                     return current_token->contract_address == address;
-                   });
+  auto token_it = base::ranges::find_if(
```

Comment:
simplify

Reply:
Unfortunately, this one in particular could not be simplified because there's an indirection in `current_token->contract_address`